paper-typeahead-input.html

```diff
@@ -543,8 +574,29 @@
       else{
         this._suggestions = matched;
       }
+    },
+    _clearSuggestings:function() {
```

Comment:
I would suggest to rename it to be _clearSuggestions to be consistent with other variables used in the code. And also as the name indicated, it should only clear the suggestions, but not make the input lose focus. So maybe create another _ function to only perform lose focus and call it in e.which = 27.   P.S.  I tried with twitter typeahead.js when clicks on ESC the input don't lose focus. But when I try with google.com or Google drive, the input loses focus when clicks on ESC. I don't know yet what's the better behavior here.